go_modules/spec/dependabot/go_modules/file_updater/go_mod_updater_spec.rb

```diff
@@ -9,12 +9,7 @@
   let(:updater) do
     described_class.new(
       dependencies: [dependency],
-      credentials: [{
-        "type" => "git_source",
-        "host" => "github.com",
-        "username" => "x-access-token",
-        "password" => "token"
-      }],
+      credentials: empty_credentials,
```

Comment:
It's more realistic to have empty or valid credentials here. Fake credentials will result in `PrivateSourceAuthenticationFailure` in some of the cases. We have a test for that in another file but I think we could move those cases over here as a future cleanup.